init.fish

```diff
@@ -1,3 +1,8 @@
+command -sq emacs
+if ! test $status -eq 0
+  exit 0
+end
```

Comment:
Why not simplify to
```fish
if not command -sq emacs
  return
end
```

Reply:
Thanks for your review!
I modified and rebased.